### PR TITLE
Update UT3HellbenderRearGun.uc Damage Type

### PR DESCRIPTION
**Pull request by HellDragon-HK**
Ok let's try this again

### CODE REVIEW

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
+	abstract;
+
+defaultproperties
+{
+     VehicleClass=Class'UT3Vehicles.UT3HellbenderRearGunPawn'
+     DeathString="%o was incinerated by %k's twin beams."
+     FemaleSuicide="%o somehow managed to shoot herself with a turret."
+     MaleSuicide="%o somehow managed to shoot himself with a turret."
+	 VehicleDamageScaling=0.85
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_HellbenderRearGunBeam.uc]: Use spaces instead of tab here.

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
+	abstract;
+
+defaultproperties
+{
+     VehicleClass=Class'UT3Vehicles.UT3HellbenderRearGunPawn'
+     DeathString="%o was incinerated by %k's twin beams."
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_HellbenderRearGunBeam.uc]: This is not quite the message you get in UT3. Check what it is there or look at the contents of UT3 `.int` files, searching for `KILLS_HELLBENDERPRIMARY`, `DEATHS_HELLBENDERPRIMARY` and `SUICIDES_HELLBENDERPRIMARY` (see http://greatemerald.xmpcommunity.com/uncodex/UT3-15/Source_utgamecontent/utdmgtype_hellbenderprimary.html).

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
+	abstract;
+
+defaultproperties
+{
+     VehicleClass=Class'UT3Vehicles.UT3HellbenderRearGunPawn'
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_HellbenderRearGunBeam.uc]: Do not include `UT3Vehicles.`, it is not needed.

File: Classes/UT3HellbenderRearGun.uc [hunk 4]
```diff
@@ -155,6 +155,7 @@ defaultproperties
     GunnerAttachmentBone = MainTurretYaw
     FireSoundClass=sound'UT3A_Vehicle_Hellbender.Sounds.A_Vehicle_Hellbender_TurretFire01'
     //BeamEffectClass=class'ShockBeamEffect'//'ONSChargeBeamEffect'
+    DamageType=Class'UT3Vehicles.UT3DmgType_HellbenderRearGunBeam'
```
Comment (GreatEmerald) [inline Classes/UT3HellbenderRearGun.uc]: Do not include `UT3Vehicles.` here either.

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc (hunk 2 above)
Comment (HellDragon-HK) [reply Classes/UT3DmgType_HellbenderRearGunBeam.uc]: I have no clue where to find them, I've tried looking on your Uncode page but can't find anything, and where would the int files be?  (in-game console doesn't work either as I'm not seeing any death messages on weapon/vehicle kills, just team switch, orb carrier, and other specials)

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_HellbenderRearGunBeam.uc]: Add the copyright notice and extend the UT2004 beam damage type class.

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
+	abstract;
+
+defaultproperties
+{
+     VehicleClass=Class'UT3HellbenderRearGunPawn'
+     DeathString="%o was disintegrated by %k's Hellbender turret beam."
+     FemaleSuicide="%o somehow managed to disintegrate herself with a Hellbender turret."
+     MaleSuicide="%o somehow managed to disintegrate himself with a Hellbender turret."
```
Comment (GreatEmerald) [inline Classes/UT3DmgType_HellbenderRearGunBeam.uc]: I'd rather these were stock UT3 messages. Sure, they may not be quite as precise, but I'd rather stick with showing off how they actually look in UT3.

Comment (HellDragon-HK) [reply Classes/UT3DmgType_HellbenderRearGunBeam.uc]: Alright, what about the more lame suicide though, UT2004 is more interesting on that most of the time

Comment (HellDragon-HK) [reply Classes/UT3DmgType_HellbenderRearGunBeam.uc]: I mean really how do you shoot yourself with a turret?  For that matter a death message in UT2004 I really love but have never seen in-game is the warp space-time and sniped them self now that's funny and thoughtful

**Comment by HellDragon-HK** [comment]
I'm unable to test the back turret much because of this which I really need some help with if you can

 - GAME PLAY RELATED
  - [ ] While only person in UT2004 Hellbender and in the back turret the vehicle de-spawned and instantly killed me with a message saying I ran over myself....I was able to replicate this on my 2nd try, it required driving away from the spawn point and I was firing the turret when it de-spawned...easily replicated, this is a serious issue

It will always happen if the vehicle is far enough from it's spawn

File: Classes/UT3DmgType_HellbenderRearGunBeam.uc (hunk 6 above)
Comment (GreatEmerald) [reply Classes/UT3DmgType_HellbenderRearGunBeam.uc]: Keep to the UT3 ones. Indeed it's pretty much impossible to suicide like that, hence keep the original wording.

**Comment by GreatEmerald** [comment]
Looks fine now.
I guess I can take a look at the possible cause of that, but it's for the bug discussion thread.

**Pull request merged**